src/ipx.ts

```diff
@@ -191,32 +189,57 @@ class IPX {
     }
 
     // Read buffer from input
-    let data = await this.get(info.src, info.adapter)
-    let sharp = Sharp(data)
-
-    if (!this.skipOperations(info)) {
-      sharp = this.applyOperations(sharp, info)
-      data = await sharp.toBuffer()
-    }
-
-    if (info.format.match(/\.json$/)) {
-      const metadata = await sharp.metadata()
-      data = Buffer.from(JSON.stringify({
-        data: `data:image/${metadata.format};base64,${data.toString('base64')}`,
-        width: metadata.width,
-        height: metadata.height,
-        size: metadata.size
-      }))
+    const originalBuffer = await this.get(info.src, info.adapter)
+    let sharp = Sharp(originalBuffer)
+    let result = null
+    const operationsContext: any = {}
+
+    switch (info.format) {
+      case 'sqip':
+        result = await this.generateSQIP(originalBuffer)
```

Comment:
Shouldn't we apply `sqip` on `sharp` result? Currently other operations will be simply dropped